### DISCUSSION TITLE
How can I change the obs no signal image (Virtual Cam)

### DISCUSSION THREAD

**Discussion by ghost** [obsproject/obs-studio#4197] [Development Questions] [answered]
How can I change the obs no signal image (Virtual Cam)

**Comment by mufunyo** [comment, marked as answer]
The no signal image is located here:
`C:\Program Files\obs-studio\data\obs-plugins\win-dshow\placeholder.png`

**Reply by ghost** [reply]
@mufunyo thanks it worked

**Reply by Meetsch** [reply]
Only issue is that the image is overwritten each time we update OBS, it would be great to have a profile setting to set the image.

**Reply by Meetsch** [reply]
Here is the idea to vote on
https://ideas.obsproject.com/posts/1325/add-setting-to-replace-placeholder-png
